integration-tests/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "runtime-integration-tests"
-version = "0.2.7"
+version = "0.3.0"
```

Comment:
why need of changes in Cargo toml for integrations tests ? it seems that the pallet is not used in the code